### PR TITLE
refactor(global): move images to zoolander

### PR DESCRIPTION
**Pull request by amberRrucker**
RSWEB-9452
Porting images from cdn to zoolander.
Effects a large number of pages, recommend full testing.

Removed some unused legacy code from some sandbox things to avoid moving over images we are no longer using.

### CODE REVIEW

File: styleguide/_themes/derek/scss/snowflakes/o365.scss [hunk 1]
```diff
@@ -173,7 +173,7 @@ a {
 }
 
 .rocket {
-  background-image: url('https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/office-365/rocketBG.png');
+  background-image: url('../imgs/derek/rocketBG.png');
```
[the remark this answers is not in the document]
Comment (amberRrucker) [reply styleguide/_themes/derek/scss/snowflakes/o365.scss]: This used to be on the o365 overview page. It's in zoolander. I guess the EBPs changed that section around so we can kill the styling on it now I presume. But best to check with Mike first to make sure it won't make a come back.

File: styleguide/css/main.scss [hunk 2]
```diff
@@ -10,7 +10,7 @@ $medium-gray: #666;
 }
 
 .siteHeader-logoImg {
```
[the remark this answers is not in the document]
Comment (amberRrucker) [reply styleguide/css/main.scss]: This is the rackspace logo in the top of the zoolander navigation. It's only used in the styleguide. It's not in www anywhere.

File: styleguide/_themes/derek/scss/snowflakes/o365.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (WidgetsBurritos) [reply styleguide/_themes/derek/scss/snowflakes/o365.scss]: I'd suggest not killing it right now.  Seems like lower risk of breaking anything. We can create a followup to kill that stuff.

File: styleguide/css/main.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (bighappyface) [reply styleguide/css/main.scss]: Thanks for the info

File: styleguide/_themes/derek/scss/snowflakes/o365.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (bighappyface) [reply styleguide/_themes/derek/scss/snowflakes/o365.scss]: Thanks for the info. No need to remove it now.

**Pull request merged**